pkg/kube/client.go

```diff
@@ -628,3 +645,66 @@ func (c *Client) watchPodUntilComplete(timeout time.Duration, info *resource.Inf
 
 	return err
 }
+
+//get an kubernetes resources's relation pods
+// kubernetes resource used select labels to relate pods
+func (c *Client) getSelectRelationPod(info *resource.Info, objPods map[string][]api.Pod) (map[string][]api.Pod, error) {
+	if info == nil {
+		return objPods, nil
+	}
+
+	err := info.Get()
```

Comment:
This can be removed one the call is moved into the `perform` function

Reply:
I moved  the call of getSelectRelationPod function perform.

Reply:
You won't need this here any more now that you have moved the call above